Update our GSOC page for 2019

ericholscher opened the pull request:
This adds a new idea for PR building.

We should also add other ideas that we've had,
especially if they are ~3-6 months out on the roadmap

davidfischer commented:
Here's some thoughts:

- Should we remove APIv3 since @humitos is working on it actively?
- I don't know if it is a great project but if we can attract any designers, I'd love to commission a Sphinx2 theme. Sphinx2 is going to be all HTML5 so a modern web framework could be used.
- This is not well thought through, but getting more different types of docs (on top of Sphinx/MkDocs) would be very cool

humitos commented on docs/gsoc.rst:
```diff
@@ -164,16 +140,33 @@ We should rebuild how we handle redirects across a number of cases:
+This would include:
+
+* Modeling Pull Requests as a type of version alongside Tags and Branches
+* Modifying how we upload HTML docs to store them in a place like S3 for long term storage
```
Long term storage for docs from a PR?

ericholscher replied:
Yea, we really don't want them on our web servers which are already space limited.

ericholscher commented:
I've removed apiv3 and added a Sphinx v2 theme section, but it isn't great. Will merge this today because of the application deadline, but if people have concrete additions, please propose them in new PR's

ericholscher commented:
@rtfd/core Would love a 👍 today if folks think this is ready.

stsewd commented on docs/gsoc.rst:
```diff
@@ -132,48 +155,20 @@ It would be great to have wrappers for the following as a start:
+More info here: 
 
+* https://github.com/rtfd/readthedocs.org/issues/1340
+* https://github.com/rtfd/readthedocs.org/issues/2465
```
These links should go in the pull requests project
